src/constants/examples.constant.ts

```diff
@@ -20,4 +20,9 @@ export const EXAMPLES = {
   LOAN_BILLING_FREQUENCY_TYPE: 'C',
   INITIATING_USERNAME: 'user1',
   FACILITY_TRANSACTION_TYPE_CODE: 2340,
+  INDUSTRY_CLASSIFICATION: '0001',
+  PARTY_NAME: 'ACTUAL IMPORT EXPORT',
+  SME_TYPE: '70',
+  CITIZENSHIP_CLASS: '2',
+  PARTY_ALTERNATE_ID: '12345678',
```

Comment:
It's not a big deal but can we use `00345678` instead please? The id's from the /numbers endpoint so will have the same prefix as DEAL_ID and FACILITY_ID